auth-web/src/components/auth/mixins/NextPageMixin.vue

```diff
@@ -200,9 +205,25 @@ export default class NextPageMixin extends Vue {
       /** If user is in the account freeze page while switching the account, then need to redirect them to account info page if that account is active.
        * otherwise user will stuck on the account freeze page **/
       if (this.$route.name?.search('account-freeze') > -1) {
+        this.anyPendingRedirect = true
         this.$router.push(`${Pages.MAIN}/${this.currentOrganization.id}/${Pages.ACCOUNT_SETTINGS}`)
       }
     }
   }
+
+  protected accountPendingRedirect () {
+    if (this.needMissingBusinessDetailsRedirect) {
+      this.anyPendingRedirect = true
```

Comment:
Maybe an enum be easier for troubleshooting?

Reply:
I didnt get this comment, This is a variable to avoid the second redirect, even if it goes pending pages.